config/application.rb

```diff
@@ -1,17 +1,16 @@
 require_relative "boot"
 
-require "rails"
 # Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
-require "active_record/railtie"
-require "active_storage/engine"
+require "action_cable/engine"
 require "action_controller/railtie"
+# require "action_mailbox/engine"
 require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
+# require "action_text/engine"
 require "action_view/railtie"
-require "action_cable/engine"
+# require "active_job/railtie"
```

Comment:
Are we going to forget that we turned off ActiveJob when we need it later?

Reply:
Possibly! Since we're likelier to use AJ over these other components, shall I flip that one on now?

Reply:
@justinlittman Done.